docs/frameworks/nextjs.md

```diff
@@ -16,9 +18,46 @@ Next.js is a React framework that allows you to build static and server-rendered
 Get started in no time with the [Next.js template](https://app.genez.io/nextjs-getting-started).
 :::
 
-## Deployment
-
-To deploy a Next.js application with Genezio, all you need to do is to run the following command in the root directory of your project:
+# Deployment
+
+Learn how to deploy an existing Next.js app using Genezio, a serverless deployment platform that simplifies app management and reduces costs
+
+### 1. Install genezio
+
+Use your preferred package manager to install Genezio:
+
+<Tabs>
+  <TabItem className="tab-item" value="npm" label="npm">
+<div id="step1-install-npm">
+  ```
+  npm install genezio -g
+  ```
+  </div>
+  </TabItem>
+  <TabItem className="tab-item" value="pnpm" label="pnpm">
+  <div id="step1-install-pnpm">
+  ```
+  pnpm add -g genezio
+  ```
+  </div>
+  </TabItem>
+  <TabItem  className="tab-item" value="yarn" label="yarn">
+  <div id="step1-install-yarn">
+  ```
+  yarn add global genezio
+  ```
+  </div>
+  </TabItem>
+</Tabs>
+
+### 2. Deploy your project
+
+:::info
+If you don't already have a Next.js app, you can create one by running in your terminal:
+```
+npx create-next-app@latest
+```
+:::
```

Comment:
I think it would be nicer here to use `genezio create` to start a NextJs template that's already Genezio compatible.

You can inline a `genezio create` command to avoid the interactive wizard with something like `genezio create --region us-east-1 --template nextjs` (this is not the exact command, but you get the idea).